src/go/cmd/token-vendor/api/v1/v1.go

```diff
@@ -389,6 +428,7 @@ func Register(tv *app.TokenVendor, prefix string) error {
 	http.HandleFunc(path.Join(prefix, "public-key.publish"), h.publicKeyPublishHandler)
 	http.HandleFunc(path.Join(prefix, "token.oauth2"), h.tokenOAuth2Handler)
 	http.HandleFunc(path.Join(prefix, "token.verify"), h.verifyTokenHandler)
+	http.HandleFunc(path.Join(prefix, "jwt.verify"), h.verifyJWTHandler)
```

Comment:
Do you need to extend https://github.com/googlecloudrobotics/core/blob/5929ab84000fb3c5d3e159686e2dafa4d6f02836/src/app_charts/token-vendor/cloud/ingress.yaml#L60?

Reply:
Yes.